README.MD

```diff
@@ -531,6 +531,32 @@ The according swagger definition can be merge-overwritten inside `tsoa.json`. He
 
 Now that you have a swagger spec (swagger.json), you can use all kinds of amazing tools that [generate documentation, client SDKs, and more](http://swagger.io/).
 
+## Decorators
+
+### Security
+
+The `Security` decorator can be used above controller methods to indicate that there should be authentication before running those methods. As described above, the authentication is done in a file that's referenced in tsoa's configuration. When using the `Security` decorator, you can choose between having one or multiple authentication methods. If you choose to have multiple authentication methods, you can choose between having to pass one of the methods (OR):
+
+```ts
+@Security('tsoa_auth', ['write:pets', 'read:pets'])
+@Security('api_key')
+@Get('OauthOrAPIkey')
+public async GetWithOrSecurity(@Request() request: express.Request): Promise<any> {
+}
+```
+
+or having to pass all of them (AND):
+
+```ts
+@Security({
+  tsoa_auth: ['write:pets', 'read:pets'],
+  api_key: [],
+})
+@Get('OauthAndAPIkey')
+public async GetWithAndSecurity(@Request() request: express.Request): Promise<any> {
+}
+```
+
```

Comment:
Documentation becomes highly required as the difference is really subtle between _AND_ and _OR_ combinations. One could really get confused without it. Good point.